src/cavendish_particle_tracks/_widget.py

```diff
@@ -40,9 +41,18 @@
 class ParticleTracksWidget(QWidget):
     """Widget containing a simple table of points and track radii per image."""
 
-    def __init__(self, napari_viewer: napari.viewer.Viewer):
+    @property
+    def layer_measurements(self):
+        return self._layer_measurements
+
+    @layer_measurements.setter
+    def layer_measurements(self, layer):
+        self._layer_measurements = layer
+
+    def __init__(self, napari_viewer: napari.viewer.Viewer, test_mode=False):
         super().__init__()
         self.viewer = napari_viewer
+        self.test_mode = test_mode
```

Comment:
That's not nice. Why do you need that?

Reply:
The UI elements are disabled until the user loads data. Tests will fail as they aren't able to access the disabled/invisible UI elements, so test mode bypasses this check that the user has loaded some data.